test/test.js

```diff
@@ -1,14 +1,14 @@
 import test from 'ava';
 import isPlainObj from 'is-plain-obj';
 import eslint from 'eslint';
-import tempWrite from 'temp-write';
+import path from 'path';
 
 const fixture = `'use strict';\nconst x = true;\n\nif (x) {\n  console.log();\n}\n`;
 
 function runEslint(str, conf) {
 	const linter = new eslint.CLIEngine({
 		useEslintrc: false,
-		configFile: tempWrite.sync(JSON.stringify(conf))
+		configFile: path.join(__dirname, conf)
```

Comment:
Why this change? It was more elegant when you could just pass the config as an object.

Reply:
I really wasn't able to get that to work on eslint 6. Because the file created was located in the "temp" directory `/temp/something` it couldn't find the node_modules folder to use the eslint configs listed under extends.